docs/examples/gameofLife.md

```diff
@@ -22,13 +22,13 @@ Here's a program that simulates cell life in the LED matrix. Use button ``A`` fo
 let lifeChart: Image = null
 
 //Use button A for the next iteration of game of life
-input.onButtonPressed(Button.A, () => {
+input.input.onButtonEvent(Button.A, ButtonEvent.Click, () => {
```

Comment:
double input?

Reply:
no, the other one is the outdated file, so this is what the kids will deal with.

Reply:
They have to type Input twice?

Reply:
ah, sorry. Yes, that's wrong. I`ll change that.